docs/support/tutorials/hpc-quick.md

```diff
@@ -0,0 +1,62 @@
+# Getting started with supercomputing
+
+You have signed up for your CSC account and first project, and are now ready to
+scale up your computations! This page is intended to give you some idea of what
+to expect from your journey into the exciting world of high-performance
+computing.
+
+## Which supercomputer to use?
+
+Projects may have access to multiple supercomputers. New users are recommended
+to start working on the Puhti supercomputer, which caters to a wide range of
+use cases. In time, you may find that you need more extensive parallelization
+options, in which case you may wish to run jobs on Mahti.
```

Comment:
```suggestion
options, in which case you may wish to run jobs on Mahti or LUMI.
```
?

Reply:
The supercomputer section is now more extensive and includes LUMI